components/vf-form/vf-form__input/vf-form__input.scss

```diff
@@ -139,37 +139,11 @@
 * modify it as you wish!
 */
 
-.vf-form__item.floatLabel {
+.vf-form__item.vf-form__floatlabel {
   position: relative;
 }
-.floatLabel {
+.vf-form__floatlabel {
   .vf-form__label {
     position: absolute;
   }
 }
-
```

Comment:
where's this gone?

Reply:
Dropped this bit as it wasn't actually being used -- or is there a plan to use it?

Reply:
ah, that's true -- liberal copying and pasting happened here :D

Reply:
![hoarding](https://media.tenor.co/images/64be5e88526f9a386e1a42f35eab9cdf/tenor.gif) :D